/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/torch/cuda/__init__.py:63: FutureWarning: The pynvml package is deprecated. Please install nvidia-ml-py instead. If you did not install pynvml directly, please report this to the maintainers of the package that installed pynvml for you.
  import pynvml  # type: ignore[import]
/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:111: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
INFO 10-01 22:44:48 [__init__.py:216] Automatically detected platform cuda.
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:44:49 [api_server.py:1896] vLLM API server version 0.10.2
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:44:49 [utils.py:328] non-default args: {'host': '0.0.0.0', 'port': 24002, 'chat_template': 'llava15_vicuna.jinja', 'model': '/home/stewa/ai-models/weights/TheBloke/llava-v1.5-13B-AWQ', 'trust_remote_code': True, 'max_model_len': 2048, 'enforce_eager': True, 'served_model_name': ['llava-1.5-13b-awq'], 'gpu_memory_utilization': 0.78}
[1;36m(APIServer pid=1184122)[0;0m The argument `trust_remote_code` is to be used with Auto classes. It has no effect here and is ignored.
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:44:52 [__init__.py:742] Resolved architecture: LlamaForCausalLM
[1;36m(APIServer pid=1184122)[0;0m `torch_dtype` is deprecated! Use `dtype` instead!
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:44:52 [__init__.py:1815] Using max model len 2048
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:44:54 [_ipex_ops.py:16] Import error msg: No module named 'intel_extension_for_pytorch'
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:44:54 [awq_marlin.py:117] The model is convertible to awq_marlin during runtime. Using awq_marlin kernel.
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:44:54 [scheduler.py:222] Chunked prefill is enabled with max_num_batched_tokens=2048.
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:44:54 [__init__.py:3400] Cudagraph is disabled under eager mode
/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/torch/cuda/__init__.py:63: FutureWarning: The pynvml package is deprecated. Please install nvidia-ml-py instead. If you did not install pynvml directly, please report this to the maintainers of the package that installed pynvml for you.
  import pynvml  # type: ignore[import]
/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/transformers/utils/hub.py:111: FutureWarning: Using `TRANSFORMERS_CACHE` is deprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
INFO 10-01 22:44:56 [__init__.py:216] Automatically detected platform cuda.
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:44:58 [core.py:654] Waiting for init message from front-end.
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:44:58 [core.py:76] Initializing a V1 LLM engine (v0.10.2) with config: model='/home/stewa/ai-models/weights/TheBloke/llava-v1.5-13B-AWQ', speculative_config=None, tokenizer='/home/stewa/ai-models/weights/TheBloke/llava-v1.5-13B-AWQ', skip_tokenizer_init=False, tokenizer_mode=auto, revision=None, tokenizer_revision=None, trust_remote_code=True, dtype=torch.float16, max_seq_len=2048, download_dir=None, load_format=auto, tensor_parallel_size=1, pipeline_parallel_size=1, data_parallel_size=1, disable_custom_all_reduce=False, quantization=awq_marlin, enforce_eager=True, kv_cache_dtype=auto, device_config=cuda, decoding_config=DecodingConfig(backend='auto', disable_fallback=False, disable_any_whitespace=False, disable_additional_properties=False, reasoning_backend=''), observability_config=ObservabilityConfig(show_hidden_metrics_for_version=None, otlp_traces_endpoint=None, collect_detailed_traces=None), seed=0, served_model_name=llava-1.5-13b-awq, enable_prefix_caching=True, chunked_prefill_enabled=True, use_async_output_proc=True, pooler_config=None, compilation_config={"level":0,"debug_dump_path":"","cache_dir":"","backend":"","custom_ops":[],"splitting_ops":null,"use_inductor":true,"compile_sizes":[],"inductor_compile_config":{"enable_auto_functionalized_v2":false},"inductor_passes":{},"cudagraph_mode":0,"use_cudagraph":true,"cudagraph_num_of_warmups":0,"cudagraph_capture_sizes":[],"cudagraph_copy_inputs":false,"full_cuda_graph":false,"pass_config":{},"max_capture_size":0,"local_cache_dir":null}
[1;36m(EngineCore_DP0 pid=1184292)[0;0m WARNING 10-01 22:44:58 [interface.py:391] Using 'pin_memory=False' as WSL is detected. This may slow down the performance.
[W1001 22:44:58.620739616 ProcessGroupNCCL.cpp:981] Warning: TORCH_NCCL_AVOID_RECORD_STREAMS is the default now, this environment variable is thus deprecated. (function operator())
[Gloo] Rank 0 is connected to 0 peer ranks. Expected number of connected peer ranks is : 0
[Gloo] Rank 0 is connected to 0 peer ranks. Expected number of connected peer ranks is : 0
[Gloo] Rank 0 is connected to 0 peer ranks. Expected number of connected peer ranks is : 0
[Gloo] Rank 0 is connected to 0 peer ranks. Expected number of connected peer ranks is : 0
[Gloo] Rank 0 is connected to 0 peer ranks. Expected number of connected peer ranks is : 0
[Gloo] Rank 0 is connected to 0 peer ranks. Expected number of connected peer ranks is : 0
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:44:59 [parallel_state.py:1165] rank 0 in world size 1 is assigned as DP rank 0, PP rank 0, TP rank 0, EP rank 0
[1;36m(EngineCore_DP0 pid=1184292)[0;0m WARNING 10-01 22:44:59 [topk_topp_sampler.py:69] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:44:59 [gpu_model_runner.py:2338] Starting to load model /home/stewa/ai-models/weights/TheBloke/llava-v1.5-13B-AWQ...
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:44:59 [gpu_model_runner.py:2370] Loading model from scratch...
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:44:59 [cuda.py:362] Using Flash Attention backend on V1 engine.
[1;36m(EngineCore_DP0 pid=1184292)[0;0m Loading safetensors checkpoint shards:   0% Completed | 0/1 [00:00<?, ?it/s]
[1;36m(EngineCore_DP0 pid=1184292)[0;0m Loading safetensors checkpoint shards: 100% Completed | 1/1 [00:04<00:00,  4.46s/it]
[1;36m(EngineCore_DP0 pid=1184292)[0;0m Loading safetensors checkpoint shards: 100% Completed | 1/1 [00:04<00:00,  4.46s/it]
[1;36m(EngineCore_DP0 pid=1184292)[0;0m
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:45:03 [default_loader.py:268] Loading weights took 4.53 seconds
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:45:04 [gpu_model_runner.py:2392] Model loading took 6.8147 GiB and 5.403826 seconds
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:45:05 [gpu_worker.py:298] Available KV cache memory: 5.32 GiB
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:45:05 [kv_cache_utils.py:864] GPU KV cache size: 6,960 tokens
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:45:05 [kv_cache_utils.py:868] Maximum concurrency for 2,048 tokens per request: 3.40x
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:45:06 [gpu_worker.py:391] Free memory on device (14.66/15.99 GiB) on startup. Desired GPU memory utilization is (0.78, 12.47 GiB). Actual usage is 6.81 GiB for weight, 0.32 GiB for peak activation, 0.02 GiB for non-torch memory, and 0.0 GiB for CUDAGraph memory. Replace gpu_memory_utilization config with `--kv-cache-memory=5555716464` to fit into requested memory, or `--kv-cache-memory=7903593472` to fully utilize gpu memory. Current kv cache memory in use is 5713002864 bytes.
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:45:06 [core.py:218] init engine (profile, create kv cache, warmup model) took 1.41 seconds
[1;36m(EngineCore_DP0 pid=1184292)[0;0m INFO 10-01 22:45:06 [__init__.py:3400] Cudagraph is disabled under eager mode
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [loggers.py:142] Engine 000: vllm cache_config_info with initialization after num_gpu_blocks is: 435
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [async_llm.py:180] Torch profiler disabled. AsyncLLM CPU traces will not be collected.
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [api_server.py:1692] Supported_tasks: ['generate']
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] Using supplied chat template: {# llava15_vicuna.jinja #}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {% set image_token = "<image>" %}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {% for m in messages %}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {% if m['role'] == 'system' %}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] SYSTEM: {{ m['content'] | trim }}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {% elif m['role'] == 'user' %}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] USER:
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {%- if m['content'] is string -%}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712]   {{ m['content'] | trim }}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {%- else -%}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712]   {# handle OpenAI Vision content blocks #}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712]   {%- for part in m['content'] -%}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712]     {%- if part['type'] == 'text' -%}{{ part['text'] | trim }}{% endif -%}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712]     {%- if part['type'] == 'image_url' -%} {{ image_token }} {% endif -%}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712]   {%- endfor -%}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {%- endif -%}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {% elif m['role'] == 'assistant' %}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] ASSISTANT: {{ m['content'] | trim }}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {% endif %}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] {% endfor %}
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] ASSISTANT:
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712]
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:45:06 [api_server.py:1712] It is different from official chat template '/home/stewa/ai-models/weights/TheBloke/llava-v1.5-13B-AWQ'. This discrepancy may lead to performance degradation.
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [api_server.py:1971] Starting vLLM API server 0 on http://0.0.0.0:24002
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:36] Available routes are:
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /openapi.json, Methods: GET, HEAD
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /docs, Methods: GET, HEAD
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /docs/oauth2-redirect, Methods: GET, HEAD
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /redoc, Methods: GET, HEAD
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /health, Methods: GET
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /load, Methods: GET
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /ping, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /ping, Methods: GET
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /tokenize, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /detokenize, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/models, Methods: GET
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /version, Methods: GET
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/responses, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/responses/{response_id}, Methods: GET
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/responses/{response_id}/cancel, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/chat/completions, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/completions, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/embeddings, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /pooling, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /classify, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /score, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/score, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/audio/transcriptions, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/audio/translations, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /rerank, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v1/rerank, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /v2/rerank, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /scale_elastic_ep, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /is_scaling_elastic_ep, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /invocations, Methods: POST
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:06 [launcher.py:44] Route: /metrics, Methods: GET
[1;36m(APIServer pid=1184122)[0;0m INFO:     Started server process [1184122]
[1;36m(APIServer pid=1184122)[0;0m INFO:     Waiting for application startup.
[1;36m(APIServer pid=1184122)[0;0m INFO:     Application startup complete.
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:45:09 [chat_utils.py:538] Detected the chat template content format to be 'openai'. You can set `--chat-template-content-format` to override this.
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] Error in preprocessing prompt inputs
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] Traceback (most recent call last):
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   File "/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/vllm/entrypoints/openai/serving_chat.py", line 227, in create_chat_completion
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     ) = await self._preprocess_chat(
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   File "/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/vllm/entrypoints/openai/serving_engine.py", line 768, in _preprocess_chat
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     conversation, mm_data_future, mm_uuids = parse_chat_messages_futures(
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   File "/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/vllm/entrypoints/chat_utils.py", line 1419, in parse_chat_messages_futures
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     sub_messages = _parse_chat_message_content(
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   File "/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/vllm/entrypoints/chat_utils.py", line 1324, in _parse_chat_message_content
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     result = _parse_chat_message_content_parts(
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   File "/home/stewa/code/imageworks/.venv/lib/python3.12/site-packages/vllm/entrypoints/chat_utils.py", line 1200, in _parse_chat_message_content_parts
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     for part in parts:
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] pydantic_core._pydantic_core.ValidationError: 7 validation errors for ValidatorIterator
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] 0.ChatCompletionContentPartTextParam.type
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   Input should be 'text' [type=literal_error, input_value='input_text', input_type=str]
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     For further information visit https://errors.pydantic.dev/2.11/v/literal_error
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] 0.ChatCompletionContentPartImageParam.image_url
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   Field required [type=missing, input_value={'type': 'input_text', 't...Respond with one word.'}, input_type=dict]
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     For further information visit https://errors.pydantic.dev/2.11/v/missing
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] 0.ChatCompletionContentPartImageParam.type
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   Input should be 'image_url' [type=literal_error, input_value='input_text', input_type=str]
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     For further information visit https://errors.pydantic.dev/2.11/v/literal_error
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] 0.ChatCompletionContentPartInputAudioParam.input_audio
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   Field required [type=missing, input_value={'type': 'input_text', 't...Respond with one word.'}, input_type=dict]
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     For further information visit https://errors.pydantic.dev/2.11/v/missing
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] 0.ChatCompletionContentPartInputAudioParam.type
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   Input should be 'input_audio' [type=literal_error, input_value='input_text', input_type=str]
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     For further information visit https://errors.pydantic.dev/2.11/v/literal_error
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] 0.File.file
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   Field required [type=missing, input_value={'type': 'input_text', 't...Respond with one word.'}, input_type=dict]
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     For further information visit https://errors.pydantic.dev/2.11/v/missing
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251] 0.File.type
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]   Input should be 'file' [type=literal_error, input_value='input_text', input_type=str]
[1;36m(APIServer pid=1184122)[0;0m ERROR 10-01 22:45:09 [serving_chat.py:251]     For further information visit https://errors.pydantic.dev/2.11/v/literal_error
[1;36m(APIServer pid=1184122)[0;0m INFO:     127.0.0.1:46256 - "POST /v1/chat/completions HTTP/1.1" 400 Bad Request
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:46:41 [launcher.py:98] port 24002 is used by process psutil.Process(pid=1184122, name='python3', status='running') launched with command:
[1;36m(APIServer pid=1184122)[0;0m WARNING 10-01 22:46:41 [launcher.py:98] /home/stewa/code/imageworks/.venv/bin/python3 -m vllm.entrypoints.openai.api_server --model /home/stewa/ai-models/weights/TheBloke/llava-v1.5-13B-AWQ --host 0.0.0.0 --port 24002 --served-model-name llava-1.5-13b-awq --tensor-parallel-size 1 --gpu-memory-utilization 0.78 --max-model-len 2048 --dtype auto --enforce-eager --trust-remote-code --chat-template llava15_vicuna.jinja
[1;36m(APIServer pid=1184122)[0;0m INFO 10-01 22:46:41 [launcher.py:101] Shutting down FastAPI HTTP server.
[rank0]:[W1001 22:46:41.399311725 ProcessGroupNCCL.cpp:1538] Warning: WARNING: destroy_process_group() was not called before program exit, which can leak resources. For more info, please see https://pytorch.org/docs/stable/distributed.html#shutdown (function operator())
[1;36m(APIServer pid=1184122)[0;0m INFO:     Shutting down
[1;36m(APIServer pid=1184122)[0;0m INFO:     Waiting for application shutdown.
[1;36m(APIServer pid=1184122)[0;0m INFO:     Application shutdown complete.
